rules.ini

```diff
@@ -248,7 +248,7 @@ CEF = (?:^|/)libcef\.(?:dll|so)$
 Coherent_Gameface_OR_Prysm = (?:^|/)cohtml\.windowsdesktop\.dll$
 CRIWARE[] = (?:^|/)data(?:1|000)?\.cpk$
 CRIWARE[] = \.(?:sfd|usm|adx|acb|awb)$
-cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|so|so\.4|so\.4\.5\.0)$
+cURL = curl(?:module|lib|d|-4|-ttv|-x64|64|_pluginw64_release)?\.(?:dll|exe|lib|so|so\.4|so\.4\.[0-9]+\.[0-9]+)$
```

Comment:
I didn't bother with so.3 because that version is decades old, but I supposed Debian used it for a while longer than that and it doesn't hurt so I've updated the regex to match that, too. I added a couple new tests as well.